website/docs/events/m4.md

```diff
@@ -0,0 +1,59 @@
+# Conference: Multiplier Event 4: Open Education
+
+Good education is always a challenge, even more so in the digital age, where information is readily available and the potential of automation is disrupting existing methods.
+Considering this, during the past years, at Open Education Hub, we worked on the following items:
+
+* we created on a methodology to develop and deploy open educational materials
+* we developed the infrastructure used for this and for automating items in the teaching process
+* we populated sample repositories, that we have used in the teaching process in university classes and in summer school
+
+We want to capture feedback on these items, to raise awareness and to encourage teachers and trainers to use them, as open source resources.
+
+So, on **Wednesday, September 13, 2023, 09:00-16:00 EEST**, we are organizing a multiplier event / conference to present our approach on developing and deploying digital educational materials as Open Education Resources (OER), using the open source collaborative model.
```

Comment:
```suggestion
So, on **Wednesday, September 13, 2023, 09:00-16:00 EEST**, we are hosting a multiplier event / conference to present our approach to developing and deploying digital educational materials as Open Education Resources (OER), using the open source collaborative model.
```
To avoid repeating "organising" below.